src/envoy/http/authn/authn_utils_test.cc

```diff
@@ -26,7 +26,7 @@ namespace Istio {
 namespace AuthN {
 namespace {
 
-const std::string kSecIstioAuthUserInfoHeaderKey = "sec-istio-auth-userinfo";
+const LowerCaseString kSecIstioAuthUserInfoHeaderKey("sec-istio-auth-userinfo");
```

Comment:
you can remove all these cast when calling GetJWTPayloadFromHeaders

Reply:
Done.